prime-router/src/main/kotlin/fhirengine/utils/FHIRBundleHelpers.kt

```diff
@@ -25,6 +31,86 @@ import java.util.stream.Stream
  * A collection of helper functions that modify an existing FHIR bundle.
  */
 const val conditionExtensionurl = "https://reportstream.cdc.gov/fhir/StructureDefinition/reportable-condition"
+const val conditionCodeExtensionURL = "https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code"
+
+/**
+ * Looks up a condition code for the passed in [code] (typically a test) using the given [metadata] object
+ * @param code test (or other type) code to look up a condition for
+ * @param metadata metadata containing an observation-mapping lookup table
+ * @return the condition code or null if no code was found
+ */
+private fun lookupCondition(code: Coding, metadata: Metadata): Coding? {
```

Comment:
No need to change it for now, but what do you think of a tech debt ticket to extract this out to more of a service?  Wondering if will have need to do this kind of work outside of processing a FHIR bundle.

Reply:
Tech debt ticket drafted